.changeset/smooth-moles-lay.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I think we should be able to skip the changeset if we add the tests only label

Reply:
Looks like this is required